pkg/sql/table_test.go

```diff
@@ -53,6 +53,11 @@ func TestMakeTableDescColumns(t *testing.T) {
 			sqlbase.ColumnType{SemanticType: sqlbase.ColumnType_INT},
 			true,
 		},
+		{
+			"BIGINT",
```

Comment:
while you're here, can you add one for `DOUBLE PRECISION`. We had a typo in the alias up until recently :(

Reply:
Done.